build-parent/pom.xml

```diff
@@ -132,7 +132,7 @@
         <webjar.bootstrap.version>4.6.2</webjar.bootstrap.version>
         <webjar.bootstrap-multiselect.version>0.9.15</webjar.bootstrap-multiselect.version>
         <webjar.bootstrap-icons.version>1.9.1</webjar.bootstrap-icons.version>
-        <webjar.font-awesome.version>6.2.1</webjar.font-awesome.version>
+        <webjar.font-awesome.version>6.4.2</webjar.font-awesome.version>
```

Comment:
@phillip-kruger is it normal we still have these now that the old Dev UI is gone?

Reply:
I thought I removed that. I'll have a look